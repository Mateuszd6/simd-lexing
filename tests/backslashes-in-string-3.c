struct foobar
{
    int a;
    int b;
    itn c;
    int d;
};

/* Move to the end */
int
main(int argc, char** argv)
{
    char const* foo = "And this is a very long string\\";
    char const* foo2 = "Which has backslashes and newlines to make sure it calculates\\\
line numbers and stuff. Hello cruel world!";

    long long long long long long long long long long long long
    long long long long long long long long long long long long
    long long long long long long long long long long long long
    long long long long long long long long long long long long;
};
