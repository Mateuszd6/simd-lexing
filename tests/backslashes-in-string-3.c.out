../tests/backslashes-in-string-3.c:1:1: TOK 0 (L = 6): "struct"
../tests/backslashes-in-string-3.c:1:8: TOK 0 (L = 6): "foobar"
../tests/backslashes-in-string-3.c:2:1: TOK 1 (L = 1): "{"
../tests/backslashes-in-string-3.c:3:5: TOK 0 (L = 3): "int"
../tests/backslashes-in-string-3.c:3:9: TOK 0 (L = 1): "a"
../tests/backslashes-in-string-3.c:3:10: TOK 1 (L = 1): ";"
../tests/backslashes-in-string-3.c:4:5: TOK 0 (L = 3): "int"
../tests/backslashes-in-string-3.c:4:9: TOK 0 (L = 1): "b"
../tests/backslashes-in-string-3.c:4:10: TOK 1 (L = 1): ";"
../tests/backslashes-in-string-3.c:5:5: TOK 0 (L = 3): "itn"
../tests/backslashes-in-string-3.c:5:9: TOK 0 (L = 1): "c"
../tests/backslashes-in-string-3.c:5:10: TOK 1 (L = 1): ";"
../tests/backslashes-in-string-3.c:6:5: TOK 0 (L = 3): "int"
../tests/backslashes-in-string-3.c:6:9: TOK 0 (L = 1): "d"
../tests/backslashes-in-string-3.c:6:10: TOK 1 (L = 1): ";"
../tests/backslashes-in-string-3.c:7:1: TOK 1 (L = 1): "}"
../tests/backslashes-in-string-3.c:7:2: TOK 1 (L = 1): ";"
../tests/backslashes-in-string-3.c:10:1: TOK 0 (L = 3): "int"
../tests/backslashes-in-string-3.c:11:1: TOK 0 (L = 4): "main"
../tests/backslashes-in-string-3.c:11:5: TOK 1 (L = 1): "("
../tests/backslashes-in-string-3.c:11:6: TOK 0 (L = 3): "int"
../tests/backslashes-in-string-3.c:11:10: TOK 0 (L = 4): "argc"
../tests/backslashes-in-string-3.c:11:14: TOK 1 (L = 1): ","
../tests/backslashes-in-string-3.c:11:16: TOK 0 (L = 4): "char"
../tests/backslashes-in-string-3.c:11:20: TOK 1 (L = 1): "*"
../tests/backslashes-in-string-3.c:11:21: TOK 1 (L = 1): "*"
../tests/backslashes-in-string-3.c:11:23: TOK 0 (L = 4): "argv"
../tests/backslashes-in-string-3.c:11:27: TOK 1 (L = 1): ")"
../tests/backslashes-in-string-3.c:12:1: TOK 1 (L = 1): "{"
../tests/backslashes-in-string-3.c:13:5: TOK 0 (L = 4): "char"
../tests/backslashes-in-string-3.c:13:10: TOK 0 (L = 5): "const"
../tests/backslashes-in-string-3.c:13:15: TOK 1 (L = 1): "*"
../tests/backslashes-in-string-3.c:13:17: TOK 0 (L = 3): "foo"
../tests/backslashes-in-string-3.c:13:21: TOK 1 (L = 1): "="
../tests/backslashes-in-string-3.c:13:24: TOK 5 (L = 32): "And this is a very long string\\"
../tests/backslashes-in-string-3.c:13:57: TOK 1 (L = 1): ";"
../tests/backslashes-in-string-3.c:14:5: TOK 0 (L = 4): "char"
../tests/backslashes-in-string-3.c:14:10: TOK 0 (L = 5): "const"
../tests/backslashes-in-string-3.c:14:15: TOK 1 (L = 1): "*"
../tests/backslashes-in-string-3.c:14:17: TOK 0 (L = 4): "foo2"
../tests/backslashes-in-string-3.c:14:22: TOK 1 (L = 1): "="
../tests/backslashes-in-string-3.c:14:25: TOK 5 (L = 107): "Which has backslashes and newlines to make sure it calculates\\\
line numbers and stuff. Hello cruel world!"
../tests/backslashes-in-string-3.c:15:44: TOK 1 (L = 1): ";"
../tests/backslashes-in-string-3.c:17:5: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:17:10: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:17:15: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:17:20: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:17:25: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:17:30: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:17:35: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:17:40: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:17:45: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:17:50: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:17:55: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:17:60: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:5: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:10: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:15: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:20: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:25: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:30: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:35: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:40: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:45: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:50: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:55: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:18:60: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:5: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:10: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:15: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:20: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:25: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:30: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:35: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:40: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:45: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:50: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:55: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:19:60: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:5: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:10: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:15: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:20: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:25: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:30: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:35: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:40: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:45: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:50: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:55: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:60: TOK 0 (L = 4): "long"
../tests/backslashes-in-string-3.c:20:64: TOK 1 (L = 1): ";"
../tests/backslashes-in-string-3.c:21:1: TOK 1 (L = 1): "}"
../tests/backslashes-in-string-3.c:21:2: TOK 1 (L = 1): ";"
Parsed: 22 lines, 70 ids, 2 strings, 0 chars, 0 ints, 0 hex,   0 floats
